_data/releases.json

```
@@ -86,5 +47,44 @@
       "end": "Oct 9 2025",
       "days": "2"
     }
-  }
+  },
+  "next_nightly": {
+    "version": "25.12",
+    "ucxx_version": "0.47",
+    "cudf_dev": {
+      "start": "Sep 18 2025",
+      "end": "Nov 12 2025",
+      "days": "39"
+    },
+    "other_dev": {
+      "start": "Sep 25 2025",
+      "end": "Nov 19 2025",
+      "days": "39"
+    },
+    "cudf_burndown": {
+      "start": "Nov 13 2025",
+      "end": "Nov 19 2025",
+      "days": "5"
+    },
+    "other_burndown": {
+      "start": "Nov 20 2025",
+      "end": "Dec 3 2025",
+      "days": "8"
+    },
+    "cudf_codefreeze": {
+      "start": "Nov 20 2025",
+      "end": "Dec 9 2025",
+      "days": "12"
+    },
+    "other_codefreeze": {
+      "start": "Dec 4 2025",
+      "end": "Dec 9 2025",
+      "days": "4"
+    },
+    "release": {
+      "start": "Dec 10 2025",
+      "end": "Dec 11 2025",
+      "days": "2"
+    }
+  },
```

JSON validators won't like this extra trailing comma. It's fine to have trailing commas in Python dicts but not JSON objects.
```suggestion
  }
```